tests/integration/helpers/0_common_enable_special_serialization.xml

```diff
@@ -0,0 +1,7 @@
+<clickhouse>
+    <profiles>
+        <default>
+            <allow_special_serialization_kinds_in_output_formats>1</allow_special_serialization_kinds_in_output_formats>
```

Comment:
We are going to enable it by default in 25.11 anyway: https://github.com/ClickHouse/ClickHouse/pull/89402/

Lazy columns replication is already enabled: https://github.com/ClickHouse/ClickHouse/pull/89316